apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -67,7 +67,34 @@ $(function() {
         </div>
         <ul class="card-body list-group">
             {{#student_submission}}
-            	<li class="{{block_formatting}}" style="margin-left: {{indent}}px">{{{inner_html}}}</li>
+                {{#distractor_feedback}}
+                <li class="pl-order-blocks-pairing-indicator list-group-item-warning">
+                    <ul style="padding:0px;">
+                        <li class="" style="list-style-type: none; ">
```

Comment:
Why the empty class? Note that Bootstrap has a `list-unstyled` style that might fit your needs here.

Reply:
oh cool, I didn't know that existed. will fix.